apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/domain-step/index.tsx

```diff
@@ -62,8 +62,12 @@ const DomainStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, o
 		<LaunchStepContainer>
 			<div className="nux-launch-step__header">
 				<div>
-					<Title>{ __( 'Choose a domain', 'full-site-editing' ) }</Title>
-					<SubTitle>{ subtitleText }</SubTitle>
+					<Title data-e2e-string="Choose a domain">
+						{ __( 'Choose a domain', 'full-site-editing' ) }
+					</Title>
+					<SubTitle data-e2e-string="Free for the first year with any annual plan.">
+						{ subtitleText }
+					</SubTitle>
```

Comment:
`wp-gutenboarding-i18n-spec.js` isn't testing this Launch flow at the moment and we're also planning to remove it in favor of Focused Launch. I've included an item in https://github.com/Automattic/wp-calypso/issues/46943 to add such tests there.

Reply:
Thanks for catching this. These attributes were supposed to be added to `client/landing/gutenboarding/onboarding-block/domains/index.tsx`. Fixed in https://github.com/Automattic/wp-calypso/pull/51000/commits/14a407eb5c6c941851d5a92e8d20ac1c89e97c3a.